dev/Generated/AnimatedVisualPlayer.properties.cpp

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// DO NOT EDIT! This file was generated by CustomTasks.DependencyPropertyCodeGen
+#include "pch.h"
+#include "common.h"
+#include "AnimatedVisualPlayer.h"
+
+CppWinRTActivatableClassWithDPFactory(AnimatedVisualPlayer)
+
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_AutoPlayProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_DiagnosticsProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_DurationProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_FallbackContentProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_IsAnimatedVisualLoadedProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_IsPlayingProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_PlaybackRateProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_SourceProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_StretchProperty{ nullptr };
+
+AnimatedVisualPlayerProperties::AnimatedVisualPlayerProperties()
+{
+    EnsureProperties();
+}
+
+void AnimatedVisualPlayerProperties::EnsureProperties()
+{
+    if (!s_AutoPlayProperty)
+    {
+        s_AutoPlayProperty =
+            InitializeDependencyProperty(
+                L"AutoPlay",
+                winrt::name_of<bool>(),
+                winrt::name_of<winrt::AnimatedVisualPlayer>(),
+                false /* isAttached */,
```

Comment:
Some properties are missing their callback functions. Please add them. Otherwise contents are not loaded as expected.